osutil/disks/gpt.go

```diff
@@ -128,18 +128,24 @@ func ReadGPTHeader(device string) (GPTHeader, error) {
 	return header, nil
 }
 
-func CalculateLastUsableLBA(device string) (uint64, error) {
-	header, err := ReadGPTHeader(device)
+func CalculateLastUsableLBA(device string, sectorSize uint64) (uint64, error) {
+	header, err := ReadGPTHeader(device, sectorSize)
 	if err != nil {
 		return 0, err
 	}
 	sectors, err := blockDeviceSizeInSectors(device)
```

Comment:
could we modify this function to take the sector size as parameter?

Reply:
Well, this is basically calling `blockdev --getsz`. So if we were to change it, I think it would be better we just remove that function and only provide a function that returns the size in bytes instead. And each caller would have the responsibility to convert to number of block.

Reply:
I have pushed a commit that replaces calls to `blockdev --getsz` by calls to `blockdev --getsize64`.